src/app/views/query-runner/QueryRunner.tsx

```diff
@@ -22,6 +22,10 @@ export class QueryRunner extends Component<IQueryRunnerProps, IQueryRunnerState>
       ],
       selectedVerb: 'GET',
       sampleURL: 'https://graph.microsoft.com/v1.0/me/',
+      sampleBody: undefined,
+      headers: [{ name: '', value: '' }],
+      headerName: '',
```

Comment:
The headers state can be derived from headerName and headerValue. If part of the state can be derived it is not necessary to explicitly define it.